pallets/parachain-staking/src/migrations.rs

```diff
@@ -79,8 +87,39 @@ mod upgrade {
 				weight_reads += 1;
 				log::info!("V10 Migrating Done.");
 			}
+
+			if onchain_storage_version < StorageVersion::new(Versions::V11 as u16) {
+				log::info!(
+					"Running storage migration from version {:?} to {:?}",
+					onchain_storage_version,
+					Versions::default() as u16
+				);
+
+				let round = <Pallet<T>>::round();
+
+				// drain CollatorBlock StorageMap item
+				for (collator, blocknum) in <CollatorBlock<T>>::iter().drain() {
```

Comment:
It's totally fine, just note that we have to test it on the Krest forked chain to make sure it won't spend too much weight during the migration

Reply:
Cool, i'll keep that in mind